.drone.yml

```diff
@@ -142,6 +142,11 @@ environment:
   DOCKER_HOST: tcp://docker:2375
 
 steps:
+  - name: fetch and checkout
+    image: alpine/git
+    commands:
+      - git fetch --tags
+      - git checkout $${VERSION}
```

Comment:
This won't work if `VERSION` is e.g. `build_123`.

Reply:
Correct we should be releasing the commit SHAs now instead of build_XXX.